### PR TITLE
feat(tomorrow): include due-today goals with bumped baremin

### PR DESCRIPTION
**Pull request by narthur**
## Summary

- `buzz tomorrow` now includes goals due **today** in addition to goals due tomorrow. Previously these were filtered out, implying a due-today goal won't be due tomorrow — but if the user does nothing today, it will be.
- For due-today goals shown in the tomorrow view, the displayed baremin is bumped by one day's worth of rate. A 1/day goal currently showing `+1` in `buzz today` now shows `+2 in 1 day` in `buzz tomorrow` — directly answering "what do I need to do today to keep this off tomorrow's list?"
- Falls back to the unmodified baremin for non-numeric formats (e.g. HH:MM time goals) or when rate info is missing, so display is never broken.

## Implementation notes

- `isDueTomorrowFilterAt` now accepts goals due today OR tomorrow (end-value-reached goals still excluded).
- New `bareminByEndOfTomorrowAt` + `ratePerDay` helpers handle the baremin adjustment.
- `handleFilteredCommand` refactored to delegate to `handleFilteredCommandWithBaremin`, which threads a per-goal baremin formatter through the column-width and rendering logic. Only the tomorrow command opts into the bumping formatter; other views are unchanged.

## Test plan

- [x] Unit tests for `ratePerDay` across y/m/w/d/h and unknown units
- [x] Unit tests for `bareminByEndOfTomorrowAt` covering rate-1/day bump, weekly rate, negative baremin, due-tomorrow passthrough, nil-rate fallback, and HH:MM non-numeric fallback
- [x] Updated `TestDueFiltersSkipEndValueReached` to reflect the new filter semantics (due-today, not-reached is now tomorrow-visible)
- [ ] Manually verify `buzz tomorrow` output against a live account

Two pre-existing test failures (`TestExtractTimeSlots`, `TestExtractTimeSlotsAcrossDates`) also fail on `main` and are unrelated to this change.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **Bug Fixes**
  * Goals due today that haven't reached their target are now included in the "tomorrow" view.
  * The tomorrow listing shows an adjusted expected-value (baremin) for due-today goals, reflecting one day’s worth of progress when rate and units are available.

* **Tests**
  * Added unit tests for per-day rate conversion and baremin adjustments across due-today/due-tomorrow and edge cases.

[Review Change Stack](https://app.coderabbit.ai/change-stack/PinePeakDigital/buzz/pull/236)

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -667,9 +723,15 @@ func handleTodayCommand() {
...
 
-// handleTomorrowCommand outputs all goals that are due tomorrow
+// handleTomorrowCommand outputs all goals that are due tomorrow. Goals that
+// are already due today are included with their baremin bumped by one day's
+// rate, so the user sees the total amount they would need to do for the goal
+// to not be due tomorrow.
 func handleTomorrowCommand() {
-	handleFilteredCommand("tomorrow", isDueTomorrowFilter)
+	now := time.Now()
+	handleFilteredCommandWithBaremin("tomorrow", isDueTomorrowFilter, func(g Goal) string {
```
[the remark this answers is not in the document]
Comment (narthur) [reply main.go]: Addressed in d2171d4 — handleTomorrowCommand now wraps the filter in a closure that calls isDueTomorrowFilterAt with the captured now, so filtering and baremin formatting share the same snapshot.

File: main.go [hunk 2]
```diff
@@ -511,17 +511,73 @@ func isDueTodayFilter(g Goal) bool {
...
+		return rate / 30.0
+	case "w":
+		return rate / 7.0
+	case "d":
+		return rate
+	case "h":
+		return rate * 24.0
+	default:
+		return rate
+	}
```
[the remark this answers is not in the document]
Comment (narthur) [reply main.go:590]: Addressed in fdcd2df by the caller-side approach: bareminByEndOfTomorrowAt now guards on a new isKnownRunits helper (y/m/w/d/h) and falls back to the goal's existing Baremin when the unit isn't one we can safely convert. ratePerDay itself is unchanged.
